modules/ssh-agent-using.adoc

```diff
@@ -108,54 +97,45 @@ link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[AWS
 ====
 endif::[]
 
-ifdef::openshift-origin[]
-[NOTE]
-====
-On clusters running {op-system-first}, the SSH keys specified in the Ignition config files are written to the `/home/core/.ssh/authorized_keys.d/core` file. However, the Machine Config Operator manages SSH keys in the `/home/core/.ssh/authorized_keys` file and configures *sshd* to ignore the `/home/core/.ssh/authorized_keys.d/core` file.
-As a result, newly provisioned {product-title} nodes are not accessible using SSH until the Machine Config Operator reconciles the machine configs with the `authorized_keys` file. After you can access the nodes using SSH, you can delete the `/home/core/.ssh/authorized_keys.d/core` file.
-====
-endif::openshift-origin[]
-
 .Procedure
 
 . If you do not have an SSH key that is configured for password-less authentication
 on your computer, create one.
 For example, on a computer that uses a Linux operating system, run the
 following command:
 +
-[source,terminal]
+ifndef::openshift-origin[]
 ----
 $ ssh-keygen -t rsa -b 4096 -N '' \
     -f <path>/<file_name> <1>
 ----
-<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the new SSH key.
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+----
+$ ssh-keygen -t ed25519 -N '' \
+    -f <path>/<file_name> <1>
+----
+endif::openshift-origin[]
+
+<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key.
+Do not specify an existing SSH key, as it will be overwritten.
```

Comment:
It took me a couple seconds to digest this advice. Have you seen user run this and accidentally delete ssh keys?

```suggestion
<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key to create.
```